lib/ansible/config/manager.py

```diff
@@ -398,8 +398,8 @@ def get_config_value_and_origin(self, config, cfile=None, plugin_type=None, plug
                     origin = 'var: %s' % origin
 
                 # use playbook keywords if you have em
-                if value is None and keys and defs[config].get('keywords'):
-                    value, origin = self._loop_entries(keys, defs[config]['keywords'])
+                if value is None and keys and config in keys:
```

Comment:
I'm confused by this change, but I don't think it breaks anything since `keywords` is unused in our docs. There are some uses of `keyword` but those all appear to be commented out. I don't know what the vision was. Could break 3rd party things if they discovered it here or [here](https://github.com/ansible/ansible/blob/27c7d5bb01f15dc89021a6019c07c10a03ed7035/lib/ansible/cli/doc.py#L488)?

Reply:
To be honest I have no context about this change.  It was made 2 authors prior to me.

Reply:
Do tests pass without it? It seems unnecessary for this feature.

Reply:
I'll find out :)

Reply:
Ah, I think I'm wrong about that. I think my confusion came from most things relying heavily on direct. I'm still confused about `keywords` syntax but this looks necessary if using task_keys.

Reply:
It does seem to be required.  Removing it and running through CI caused 63 test failures: https://app.shippable.com/github/ansible/ansible/runs/105917/summary/console